R/sequential.R

```diff
@@ -0,0 +1,94 @@
+#' Perform multiple requests in sequence
+#'
+#' Given a list of requests, this function performs each in turn, returning
+#' a list of responses. It's slower than [req_perform_parallel()] but
+#' has fewer limitations.
+#'
+#' @inheritParams req_perform_parallel
+#' @inheritParams req_perform_iteratively
+#' @param on_error What should happen if one of the requests throws an
+#'   HTTP error?
+#'
+#'   * `stop`, the default: stop iterating and throw an error
+#'   * `return`: stop iterating and return all the successful responses so far.
+#'   * `ignore`: continue iterating, recording errors in the result.
```

Comment:
`on_error = "ignore"` doesn't read as nicely as with `stop` and `return`. I think `on_error = "continue"` would be a bit more clear (and you already mentioned the "continue" option above anyways).

Reply:
Agreed